src/components/Tool/Appeal/List/AppealTour/AppealTour.tsx

```diff
@@ -0,0 +1,233 @@
+  backgroundColor: color,
+  textAlign: 'center',
+  padding: `${theme.spacing(2)} ${theme.spacing(2)}`,
+  position: 'relative',
+  width: '20%',
+  '&:after': {
+    content: '" "',
+    width: '24px',
+    height: '24px',
+    position: 'absolute',
+    right: '-12px',
+    zIndex: '50',
+    background: color,
+    top: '50%',
+    transform: 'translateY(-50%) rotate(45deg)',
+  },
+  '@media (max-width: 550px)': {
+    width: '100%',
+    padding: `${theme.spacing(1)} ${theme.spacing(1)}`,
+    '&:after': {
+      width: '16px',
+      height: '16px',
+      top: '100%',
+      right: '50%',
+      transform: 'translate(50%, -50%) rotate(45deg)',
+    },
+  },
+}));
+
+type ActionButtonsProps = {
+  nextTourStep: () => void;
+  hideTour: () => void;
+  tourStep: AppealTourEnum;
+  t: TFunction;
+};
+const ActionButtons = ({
+  nextTourStep,
+  hideTour,
+  tourStep,
+  t,
+}: ActionButtonsProps) => {
+  return (
+    <Box display={'flex'} sx={{ marginTop: theme.spacing(2) }}>
+      <Box>
+        {tourStep !== AppealTourEnum.Finish && (
+          <Button
+            onClick={() => {
+              hideTour();
+            }}
+            variant="outlined"
+            sx={{ marginRight: theme.spacing(2) }}
+          >
+            {t('Hide')}
+          </Button>
+        )}
+
+        <Button
+          onClick={() => {
+            nextTourStep();
+          }}
+          variant="contained"
+        >
+          {tourStep === AppealTourEnum.Start && t('Start')}
+          {tourStep === AppealTourEnum.Finish && t('Finished')}
+          {tourStep === AppealTourEnum.ExportContacts && t('Export to CSV')}
+
+          {tourStep !== AppealTourEnum.Start &&
+            tourStep !== AppealTourEnum.ExportContacts &&
+            tourStep !== AppealTourEnum.Finish &&
+            t('Next')}
+        </Button>
+      </Box>
+    </Box>
+  );
+};
+
+type StatisticContainerProps = {
+  color: string;
+  statisticNumber: number;
+  statisticName: string;
+};
+const StatisticContainer = ({
+  color,
+  statisticNumber,
+  statisticName,
+}: StatisticContainerProps) => {
+  return (
+    <StatisticBox color={color}>
+      <Typography
+        variant="h4"
+        fontWeight="bold"
+        color={theme.palette.common.white}
+      >
+        {statisticNumber ?? 0}
+      </Typography>
+      <Typography
+        variant="body1"
+        fontWeight="bold"
+        color={theme.palette.common.white}
+      >
+        {statisticName ?? ''}
+      </Typography>
+    </StatisticBox>
+  );
+};
+
+export const AppealTour: React.FC = () => {
+  const { t } = useTranslation();
+  const [tourTitle, setTourTitle] = React.useState<string>('');
+  const [tourText, setTourText] = React.useState<string>('');
+  const {
+    tour,
+    hideTour,
+    nextTourStep,
+    askedCountQuery,
+    excludedCountQuery,
+    committedCountQuery,
+    givenCountQuery,
+    receivedCountQuery,
+  } = React.useContext(AppealsContext) as AppealsType;
```

Comment:
And import `useContext` from `'react'`.

```suggestion
  } = useContext(AppealsContext) as AppealsType;
```